WORKFLOWS.md

```diff
@@ -80,4 +83,71 @@ OpenSearch project releases add a [github tag](https://docs.github.com/en/rest/g
 
 1. Setup release notes directory with the repo, see OpenSearch [example](https://github.com/opensearch-project/OpenSearch/tree/main/release-notes). This workflow is dependent on predictable release notes based on a release tag.
 2. Setup the workflow in your repo similar to [auto-release.yml](https://github.com/opensearch-project/OpenSearch/blob/main/.github/workflows/auto-release.yml) in OpenSearch.
-3. Backport this workflow to all release branches i.e 2.x, 1.x etc.
\ No newline at end of file
+3. Backport this workflow to all release branches i.e 2.x, 1.x etc.
+
+## Publishing snapshots to Maven
+
+OpenSearch project uses GitHub Actions to publish snapshots to maven. 
```

Comment:
Let's link https://mvnrepository.com/artifact/org.opensearch?

Reply:
Any idea what is the snapshot equivalent for that? I know https://aws.oss.sonatype.org/content/repositories/snapshots/